python/tk_unreal/unreal_sg_engine.py

```diff
@@ -319,15 +323,15 @@ def _add_app_menu(self, commands_by_app, menu_items):
         """
         try:
             has_selected_actors = len(self.get_selected_actors()) > 0
-        except:
+        except Exception:
```

Comment:
Is ok to not log this exception?

Reply:
I don't know, I just fixed the bare except which wouldn't pass flake8.